osquery/config/plugins/tls.cpp

```diff
@@ -67,16 +75,23 @@ Status TLSConfigPlugin::setUp() {
   }
 
   uri_ = TLSRequestHelper::makeURI(FLAGS_config_tls_endpoint);
-
-  // If the initial configuration includes a non-0 refresh, start an additional
-  // service that sleeps and periodically regenerates the configuration.
-  if (FLAGS_config_tls_refresh >= 1) {
-    Dispatcher::addService(std::make_shared<TLSConfigRefreshRunner>());
-  }
+  current_delay = FLAGS_config_tls_refresh;
 
   return Status(0, "OK");
 }
 
+void TLSConfigPlugin::updateDelayPeriod(bool success) {
+  if (success) {
+    if (current_delay != FLAGS_config_tls_refresh) {
+      LOG(INFO) << "Normal configuration delay restored";
+      current_delay = FLAGS_config_tls_refresh;
+    }
+  } else {
+    LOG(WARNING) << "Using accelerated configuration delay";
```

Comment:
Will this be logging every time? I'd expect it to output the message once when accelerated configuration checks are enabled, and once again when disabled.